vendor/github.com/jacobsa/gcloud/gcs/bucket.go

```diff
@@ -350,13 +352,25 @@ func (b *bucket) DeleteObject(
 }
 
 func newBucket(
+	ctx context.Context,
 	client *http.Client,
 	url *url.URL,
 	userAgent string,
 	name string,
 	billingProject string) Bucket {
+
+	// Creating client through GSCL for the sclClient parameter of bucket.
+	var sclClient *storage.Client = nil
+	sclClient, err := storage.NewClient(ctx)
+	if err != nil {
+		err = fmt.Errorf("Error in creating the client through GSCL: %v", err)
+	} else {
+		fmt.Println("GSCL Client Created")
+	}
+
```

Comment:
we should use loggers instead of printing messages to console. Also in this case you can avoid printing a message. Its obvious that if error is not printed the client is created.

Reply:
okay. I'll remove this.